src/Datadog.Trace.AutoInstrumentation/Datadog.Trace.AutoInstrumentation.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <Version>1.28.1-prerelease</Version>
```

Comment:
```suggestion
      <Version>1.28.2</Version>
```

Reply:
Fixed in e3a1b4932e8484173637e23a68b4d78e4475b449